src/MyMap/MyHashMap.java

```diff
@@ -48,25 +57,26 @@ public void put(Object key, Object value) {
 
             } else if (capacity - size == 1) {
                 //TODO change name to something like normalizeSize
-                resize();
+                normalizeMap();
             }
 
+
             Object[] pairKeyValue = new Object[2];
             pairKeyValue[0] = key;
             pairKeyValue[1] = value;
 
+
+
             size++;
```

Comment:
better ++size because we don't use the size's value before the increment.

Reply:
ok